add-ons/capsule/values.yaml

```diff
@@ -1,4 +1,11 @@
-capsule :
+capsule:
+  customAnnotations:
+    release: capsule
+  certManager:
```

Comment:
please add comment here that we need to ensure cert-manger is installed first

Reply:
Successfully added comments to the new fields